plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java

```diff
@@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
   public void testValidProps() throws QuarkException {
     Map<String, Object> props = new HashMap<>();
     props.put("type", "mysql");
-    props.put("url", "jdbc:quark:test");
+    props.put("url", "fatjdbc:quark:test");
```

Comment:
Change that to what I suggested. Hopefully it's a global search replace
On Tue, 8 Mar 2016 at 19:28, adeshr <notifications@github.com> wrote:

> In
> plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java
> <https://github.com/qubole/quark/pull/32#discussion_r55361194>:
>
> > @@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
> >    public void testValidProps() throws QuarkException {
> >      Map<String, Object> props = new HashMap<>();
> >      props.put("type", "mysql");
> > -    props.put("url", "jdbc:quark:test");
> > +    props.put("url", "fatjdbc:quark:test");
>
> Also, connect string is "jdbc:quark:" for fatjdbc, and "jdbc:quark:thin:"
> for jdbc. I didn't make any changes in fatjdbc connect string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qubole/quark/pull/32/files#r55361194>.
>